src/test/java/com/stevedutch/assignment14/web/ChannelControllerTest.java

```diff
@@ -55,19 +48,13 @@ void myShowChannelTest() {
 		String expectedResult = sut2.showChannel(sut);
 
 		assertEquals("/channel", expectedResult);
-		assertTrue(sut.containsAttribute("user"));
+		assertFalse(sut.containsAttribute("user"));
 	}
 
-	@Mock
-	private ModelMap modelMap;
-
-	@InjectMocks
-	ChannelController underTest;
```

Comment:
no need to declare the class attributes for ModelMap and ChannelController several times throughout the class. Just once at the top is usually enough for simple tests